src/mongo/db/mongod_options.cpp

```diff
@@ -208,6 +210,16 @@ Status validateMongodOptions(const moe::Environment& params) {
         return ret;
     }
 
+    if (params.count("nojournal") || params.count("storage.journal.enabled")) {
+        LOGV2_WARNING_OPTIONS(
+            29117,
+            {logv2::LogTag::kStartupWarnings},
+            "The storage.journal.enabled option and the corresponding --journal and "
+            "--nojournal command-line options have no effect in this version of Percona "
+            "Server for MongoDB. Journaling is always enabled. Please remove those "
+            "options from the config");
```

Comment:
The last sentence in the log message (`Please remove those ...`) doesn't end with a period (`.`). It is not a big issue, though.

Reply:
The comment above should be ignored because it suggests violating the [recommendations](https://github.com/percona/percona-server-mongodb/blob/master/docs/logging.md?plain=1#L45) for writing a log message.